lib/ppx_mysql_runtime.mli

```diff
@@ -1,10 +1,9 @@
 (********************************************************************************)
-(** {1 Public functions and values}                                             *)
-(********************************************************************************)
-
-val identity: 'a -> 'a
 
-val map_option: ('a -> 'b) -> 'a option -> 'b option
+(** {1 Public functions and values}                                             *)
 
-val get_option: 'a option -> 'a
+(********************************************************************************)
```

Comment:
It concerns me a little that our formatter would be so concerned about pure comments.

Reply:
But it sort of isn't, because these are comments for the documentation generator.